docs/admin/Offline_System_Certificate_Renewal.md

```diff
@@ -6,65 +6,50 @@ Offline System Certificate Renewal
 PKI server provides a mechanism to recover from expired system certificates. This mechanism can also be
 used to renew the certificates before they expire. There are 2 ways to renew the certs
 
-1. [Automated Renewal Process](#Automated-Renewal-Process)
-2. [Manual Renewal Process](#Manual-Renewal-Process) 
+1. [Automated Renewal Process](#Automated-Renewal-Process) - supports LDAPS/LDAPI configuration
+2. [Manual Renewal Process](#Manual-Renewal-Process) - supports LDAP/LDAPS/LDAPI configuration
 
-It is assumed that you have the following:
-* Valid CA signing cert
-* Valid admin cert
+This tool's behavior is different in an **IPA environment** and **standalone PKI environment**
 
-To verify these assumptions are valid:
+## Automated Renewal Process
 
-1. List details of all system certificates. (Note down the `<cert ID>` of the certificates that need to be renewed)
+### IPA Environment (Uses LDAPI)
 
-    ````
-    # pki-server cert-find
+#### Assumptions:
 
-      Cert ID: ca_signing
-      Nickname: ca_signing
-      Serial Number: 0x1
-      Subject DN: CN=CA Signing Certificate,OU=pki-tomcat,O=EXAMPLE
-      Issuer DN: CN=CA Signing Certificate,OU=pki-tomcat,O=EXAMPLE
-      Not Valid Before: Wed Dec 19 17:33:21 2018
-      Not Valid After: Sun Dec 19 17:33:21 2038
-    ````
+- Valid CA certificate [WIP to remove this assumption]
+- `cert-fix` must be run as `root`
+- `LDAPI` must be configured, with `root` autobinding to `cn=Directory Manager` or other account with privileges on `o=ipaca` subtree, including password reset privileges
+- The password of the specified agent account will be reset. If needed, it can be changed back afterwards (manually; successful execution of `cert-fix` proves that the operator has privileges to do this)
+- If Dogtag was configured to use TLS certificate authentication to bind to LDAP, the password on the `pkidbuser` account will be reset. (If password authentication was already used, the password does not get reset)
```

Comment:
Just curious, is it possible to use cert auth over LDAPI?

Reply:
The assumption is that all system certs are expired. So, @frasertweedale came up with the solution to pass in the LDAPI socket

Reply:
This section is about IPA environment which uses LDAPI. So if using TLS certificate authentication is not applicable to LDAPI, this item probably should be removed.